notebooks/277-amused-lightweight-text-to-image/277-amused-lightweight-text-to-image.ipynb

```diff
@@ -0,0 +1,689 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.    def generate(prompt, seed):
</pre><p>We can track the progress inside the Gradio interface using <code>gr.Progress</code> as the last function argument:</p><pre class="ql-syntax" spellcheck="false">def generate(prompt, seed, _=gr.Progress(track_tqdm=True))
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1620/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='30'/>

Reply:
Thanks, added